packages/scss/src/components/_table.scss

```diff
@@ -134,3 +158,41 @@
 		}
 	}
 }
+
+
+// STATES
+// ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
+
+.table-head-row-cell {
+	&.is-sorted-descending {
+		color: _color("text.dark");
+
+		&::after {
+			@include makeIcon("arrow_full_south");
+			opacity: 1;
+			top: 0;
+		}
+
+		&:hover {
+			&::after {
+				opacity: .75;
+			}
+		}
+	}
```

Comment:
It's a bit different than action icons : This icon give a direct information on the table's state. Furthermore, clicking on it will change his state (descending > ascending).